### PR TITLE
HEEDLS-325 Change post learning page status text to green

### PR DESCRIPTION
**Pull request by benjimarshall**
## Changes
Change post learning status text on the post learning page to green
when the assessment has been completed successfully.

The text size does not need to be changed, already it is 24px on the
narrowest of displays.

## Testing
Add some unit tests, tested in Chrome, Firefox, Edge and IE11, and using a screenreader.

### Not attempted post learning assessment
not_attempted_post_learning
### Failed post learning assessment
failed_post_learning_assessment
### Completed post learning assessment
green_post_learning_status

### CODE REVIEW

File: DigitalLearningSolutions.Web/ViewModels/LearningMenu/PostLearningAssessmentViewModel.cs [hunk 1]
```diff
@@ -29,10 +30,12 @@ public PostLearningAssessmentViewModel(PostLearningAssessment postLearningAssess
             if (postLearningAssessment.PostLearningAttempts == 0)
             {
                 AssessmentStatus = "Not attempted";
+                AssessmentStatusStyling = "not-passed-text";
             }
             else
             {
                 AssessmentStatus = GetPassStatus(postLearningAssessment);
+                AssessmentStatusStyling = GetPassStatusStyling(postLearningAssessment);
```
Comment (FridaTveit) [inline DigitalLearningSolutions.Web/ViewModels/LearningMenu/PostLearningAssessmentViewModel.cs:38]: You could assign this outside the if? Saves needing to type out the not passed css class twice (which means there's a chance they could become inconsistent). And I think assigning to a variable just once makes the code easier to read (instead of assigning to it in the if branch and the else) 🙂

**Review by FridaTveit — COMMENTED** [review]
One small suggestion otherwise looks good 🙂

File: DigitalLearningSolutions.Web/ViewModels/LearningMenu/PostLearningAssessmentViewModel.cs (hunk 1 above)
Comment (benjimarshall) [reply DigitalLearningSolutions.Web/ViewModels/LearningMenu/PostLearningAssessmentViewModel.cs]: Yeah that looks better! Done in 42a3e2f 🙂

**Review by FridaTveit — APPROVED** [review]
Looks good 🙂
